checkov/terraform/checks/resource/aws/EKSControlPlaneLogging.py

```diff
@@ -20,9 +20,13 @@ def scan_resource_conf(self, conf):
         """
         log_types = ["api", "audit", "authenticator", "controllerManager", "scheduler"]
         if "enabled_cluster_log_types" in conf.keys() and conf["enabled_cluster_log_types"] and \
```

Comment:
Can you please use conf.get("enabled_cluster_log_types") once so that we don't do multiple dictionary lookups?